frontend/lib/src/components/shared/FullScreenWrapper/FullScreenWrapper.test.tsx

```diff
@@ -51,4 +55,12 @@ describe("FullScreenWrapper", () => {
     // queryBy returns null vs. error
     expect(screen.queryByTestId("StyledFullScreenButton")).not.toBeNull() // eslint-disable-line testing-library/prefer-presence-queries
   })
+
+  it("hide find StyledFullScreenButton when hideFullScreenButton is true", () => {
```

Comment:
super nit: `"hides StyledFullScreenScreenButton when disableFullscreenMode is true"`

Reply:
Updated